### PR TITLE
fix(services): fix empty state for jobs

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-921

2023-11-29-120740_1514x714_scrot

Prefer "Hide whitespace" for review

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> You are going to thank you later if it contains tests

File: libs/pages/application/src/lib/ui/page-general/page-general.tsx [hunk 1]
```diff
@@ -19,24 +19,25 @@ export function PageGeneral(props: PageGeneralProps) {
...
+                <PodsMetrics environmentId={environmentId} serviceId={applicationId}>
+                  {isCronJob && (
+                    <div className="grid grid-cols-[min-content_1fr] gap-x-3 gap-y-1 p-3 border rounded border-neutral-250 text-xs text-neutral-350 bg-neutral-100">
+                      <Icon className="row-span-2" name={IconAwesomeEnum.CIRCLE_INFO} />
+                      <p>
+                        The number of past Completed or Failed job execution retained in the history and their TTL can
+                        be customized in the advanced settings.
+                      </p>
+                      <ExternalLink
+                        className="text-xs"
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/page-general/page-general.tsx:31]: fixed https://github.com/Qovery/console/pull/998/commits/b7c8036e614c781eb7b865c02be72a667c5021b2

**Comment by ctjhoa** [comment]
> Thanks @ctjhoa !
> 
> * We have a glitch on the first load: https://www.loom.com/share/8d15d6466cb04c5fae4a370bb30d7ffc?sid=e3eb66ef-e919-4cb0-b22b-ac1e83510cde

Fixed https://github.com/Qovery/console/pull/998/commits/8c6128016729b316728b84de3f526972ec3ad7a6

**Review by RemiBonnet — APPROVED** [review]
Perfecto !

**Pull request merged**